docs/docs/data-structures/tree/reading-and-editing.mdx

```diff
@@ -77,41 +81,46 @@ The read APIs for map nodes have the same names and syntax as the corresponding
 has(key): boolean
 ```
 
-Returns `true`` if the key is present in the map.
+Returns `true` if the key is present in the map.
 
 ```typescript
 get(key): T | undefined
 ```
 
-Returns the value of the property with the specified key.
+Returns the value associated with the specified key if any, and returns `undefined` if no value is associated with that key.
 
 ```typescript
 keys(): IterableIterator<string>
 ```
 
-Returns an Iterator that contains the keys in the map node. The keys are iterated in the order that they were added.
+Returns an Iterator that contains the keys in the map node.
+We do not make guarantees about the ordering of the keys.
```

Comment:
I think specific info like this should be kept on the API docs and we should do more linking

Reply:
Yeah, agreed. I mostly wanted to stop telling lies, so I'm happy to just remove the lie and not mention the truth here (the API does say "If your usage scenario depends on consistent ordering, you will need to sort these yourself").

Reply:
Yeah, I like that approach